src/lib/collision_prevention/CollisionPrevention.cpp

```diff
@@ -105,6 +105,18 @@ hrt_abstime CollisionPrevention::getElapsedTime(const hrt_abstime *ptr)
 	return hrt_absolute_time() - *ptr;
 }
 
+bool CollisionPrevention::is_active()
+{
+	bool is_active = _param_cp_dist.get() > 0;
```

Comment:
It seems there's another variable called `is_active` that Jenkins is complaining about. So this should be renamed.